src/main/java/uk/gov/hmcts/darts/cases/controller/CaseController.java

```diff
@@ -108,10 +108,6 @@ public ResponseEntity<SingleCase> casesCaseIdGet(Integer caseId) {
 
     }
 
-    @Override
-    public ResponseEntity<List<EventResponse>> getEvents(@Parameter(name = "hearing_id", description = "hearingId is the internal hea_id of the hearing.", required = true, in = ParameterIn.PATH) @PathVariable("hearing_id") Integer hearingId) {
-        return new ResponseEntity<>(caseService.getEvents(hearingId), HttpStatus.OK);
```

Comment:
Is the `caseService.getEvents()` implementation still needed? If not, can associated code/tests can be removed from `CaseService`/`CaseServiceImpl`?